documentation/docs/guide/experiments/create-an-experiment.md

```diff
@@ -4,13 +4,13 @@ Follow the relevant steps below to create an experiment in H2O LLM Studio.
 
 1. On the H2O LLM Studio left-navigation pane, click **Create experiment**. Alternatively, you can click **New experiment** in the kebab menu of the [View datasets](../view-dataset.md) page.
 
-2. Select the dataset you want to use to fine-tune an LLM model.
+2. Select the **Dataset** you want to use to fine-tune an LLM model.
 
-3. Select the problem type.
+3. Select the **Problem type**.
 
-4. Provide a meaningful experiment name.
+4. Provide a meaningful **Experiment name**.
 
-5. Select the LLM Backbone, the foundation model that you continue training. H2O LLM Studio has a predefined list of recommended types of foundation models, but you can also use [Hugging Face models](https://huggingface.co/models). 
+5. Select the **LLM Backbone**, the foundation model that you continue training. H2O LLM Studio has a predefined list of recommended types of foundation models, but you can also use [Hugging Face models](https://huggingface.co/models). 
```

Comment:
As you suggested, let's add "LLM Backbone" to the key terms and link off to it to explain the concept.